include/nix/Block.hpp

```diff
@@ -15,7 +15,7 @@
 #include <nix/DataArray.hpp>
 #include <nix/MultiTag.hpp>
 #include <nix/Tag.hpp>
-
+#include <nix/util/util.hpp>
```

Comment:
Is that really needed? Especially since you moved the checks to the source file?